prog/vm/github_runner.rb

```diff
@@ -131,10 +131,24 @@ def before_run
   end
 
   label def wait_concurrency_limit
-    Clog.emit("Waiting for customer concurrency limit") { {github_runner: github_runner.values} }
+    unless concurrency_available?
```

Comment:
nit: early exit less indentation
```suggestion
    hop_allocate_vm if concurrency_available?
```